manywheel/Dockerfile

```diff
@@ -142,7 +142,7 @@ RUN rpm -e cmake || true
 # cmake-3.18.4 from pip
 RUN yum install -y python3-pip && \
     python3 -mpip install cmake==3.18.4 && \
-    ln -s /usr/local/bin/cmake /usr/bin/cmake
+    ln -sf /usr/local/bin/cmake /usr/bin/cmake
```

Comment:
@jeffdaily Can you please elaborate why the force link is needed as part of this change?

Reply:
The pre-build of the magma-miopen-staging image requires cmake 3 and various dockerfile targets all try to pip install cmake and create the same symlink.  When we layer these different targets the symlink already exists and will error unless forced.  The pip install will exit cleanly as already complete.